index.html

```diff
@@ -23,16 +23,25 @@
     <link rel="stylesheet" type="text/css"
     media="screen and (min-width: 48em)"
     href="desktopstyle.css" />
+    <script src="script.js" defer></script>
   </head>
-  <body>
+  <body id="Porfolio">
+    <section class="mobile-menu hidden">
+      <a href="#" onclick="hide()">
+        <img src="images/Icon - Cancel.svg" id="cancel" alt="cancel">
+      </a>
+      <a href="#Portofolio" class="m-menu marg">Portfolio</a><hr>
+      <a href="#About" onclick="hide()" class="m-menu">About</a><hr>
+      <a href="#Contact" onclick="hide()" class="m-menu">Contact</a>
```

Comment:
- [ ] You should try as much as possible to follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). I suggest that you also include the line below the contact link. This will make your portfolio have a great UI ☺️.